SEED-platform/seed
# Several miscellaneous fixes

#### What's this PR do?
- Updates copyrights to 2021
- Adds optional logging of all un-translated strings (commented by default)
- Updates translations with many missing strings
- Fixed Django's `debug` setting so that it could be passed to the UI
- Supervisor tweak so that commands like `supervisorctl stop uwsgi` will work
- Fixed maintenance page email and favicon request, added automatic timestamp

#### How should this be manually tested?
- Check that window.BE.debug accurately reflects the debug setting
- Check that enabling the maintenance page shows the timestamp when it was enabled
